projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.html

```diff
@@ -29,7 +30,9 @@
       format="product"
     >
     </cx-media>
-    <h4>{{ item.name }}</h4>
-    <div class="price">{{ item.price?.formattedValue }}</div>
+    <h4 [ngStyle]="carouselItemStyle$ | async">{{ item.name }}</h4>
```

Comment:
instead of putting them individually on h4 and div, maybe we can put it in the a tag that encapsulate these tags